tests/unit/lms/services/vitalsource/client_test.py

```diff
@@ -110,6 +195,8 @@ def pyramid_config(self, pyramid_config):
         pyramid_config.registry.settings[
             "vitalsource_lti_launch_secret"
         ] = sentinel.vs_lti_launch_secret
+        pyramid_config.registry.settings["vitalsource_api_key"] = sentinel.vs_api_key
```

Comment:
Might just want to put this in the global test settings to that it's always present: https://github.com/hypothesis/lms/blob/98e70824fce1a39e6906002ad7184e8efe20dff8/tests/conftest.py#L16-L45

Reply:
Moved there :+1: